ext/ITensorNetworksEinExprsExt/src/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,67 @@
+module ITensorNetworksEinExprsExt
+
+using ITensors
+using ITensorNetworks
+using ITensorNetworks: Index
+using EinExprs: EinExprs, EinExpr, einexpr, SizedEinExpr
+
+function EinExprs.einexpr(tn::ITensorNetwork; optimizer::EinExprs.Optimizer)
+  IndexType = Any
+  T = vertextype(tn)
+
+  tensors = EinExpr{IndexType}[]
+  tensor_map = Dict{Set{IndexType},T}()
+  sizedict = Dict{IndexType,Int}()
+
+  for (key, tensor) in pairs(vertex_data(tn))
```

Comment:
I think it is better to write this as:
```julia
for vertex in vertices(tn)
  tensor = tn[vertex]
  ...
end
```
since `vertex_data` should be considered an internal detail of the `ITensorNetwork` type.